src/__snapshots__/TextHelpers.test.js.snap

```diff
@@ -1,11 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`formatDatePair displays differences of dates 1`] = `"Mon, Feb 4, 2019 at 11:41AM MST"`;
+exports[`formatDatePair displays differences of dates 1`] = `"Mon, Feb 4, 2019, 11:41 AM CST"`;
```

Comment:
we lose the "at", but we gain internationalization.

Reply:
maybe we use @ instead of comma?